src/integrations/popupBridge.js

```diff
@@ -163,6 +165,9 @@ function renderThroughPopupBridge(props : Object, popupBridge : PopupBridge) : Z
 
         } else if (opType === OPTYPE.CANCEL) {
             return onCancel(data, actions);
+        
+        } else if (opType === OPTYPE.SHIPPING_CHANGE) {
```

Comment:
Have we opened a conversation with the teams that own the popup bridge so they can support this optype? (native sdk and braintree)?

Actually I wonder if it's even possible...